spec/models/hmcts_common_platform/defendant_summary_spec.rb

```diff
@@ -1,5 +1,20 @@
 RSpec.describe HmctsCommonPlatform::DefendantSummary, type: :model do
-  let(:defendant_summary) { described_class.new(data) }
+  let(:court_application_summary) do
+    [
+      {
+        "applicationId": "f38d0030-0b4a-4fa5-9484-bb37b1e6ab39",
+        "shortId": "A25ULRHLVC7S",
+        "reference": "MyString",
+        "title": "Appeal at conviction here",
+        "receivedDate": "2025-04-11",
+        "subjectSummary": {
+          "defendantFirstName": "Bob",
+          "defendantLastName": "Smith",
+        },
+      },
+    ]
+  end
+  let(:defendant_summary) { described_class.new(data, :court_application_summary) }
```

Comment:
This is a bug - you're not passing in `court_application_summary`, you're passing in `:court_application_summary`, i.e. just a symbol. Because of that I'm guessing `defendant_summary.application_summaries` is blank, so your test "passes" but only because it's not testing anything. I think this is a sign we need better tests here.